bip-0200.mediawiki

```diff
@@ -0,0 +1,91 @@
+<pre>
+  BIP: 200
+  Layer: Consensus (soft fork reversion)
+  Title: Mandatory de-activation of forced segwit deployment
+  Author: @movrcx <joshua2014@protonmail.ch>
+  Comments-Summary: No comments yet.
+  Comments-URI: TBD
+  Status: Draft
+  Type: Standards Track
+  Created: 2017-06-02
+  License: Public Domain (Unlicensed)
+</pre>
+
+==Abstract==
+
+This BIP supercedes BIP148 and outlines the methods and actions necessary to prevent unwanted network segmentation and forced isolation caused by non-consensual BIP148 and Segregated Witness deployment.
+
+The Bitcoin protocol was initially designed as a decentralized standard. Unfortunately BIP148 establishes and assigns significant standards-implementation authority to large and more influential commercial organizations and away from independent parties. The author of this proposal fully rejects BIP148 as being contrary to the spirit of Bitcoin and rejects the premise that overall Bitcoin consensus can be gained through a simple voting mechanism.
+
+BIP148 (UASF) additionally creates the scaffolding required for large influential groups to take ownership of the global Bitcoin protocol which is a significant departure from the original decentralized nature of Bitcoin. The presence of this scaffolding creates extraordinary risk of centralization within the Bitcoin community and must be resisted.
+
+This BIP takes a red-team-centric approach and proposes multiple legally acceptable covert and overt means to resist and eradicate BIP148 and SegWit from the global Bitcoin network in the event it is non-consensually activated. 
```

Comment:
I think you are completely mischaracterizing the SegWit debate here. 

How do you expect the network to reach consensus when the majority of hashing power opposes SegWit SF activation due to the fact that it removes their ability to utilize covert ASICBoost? Why do you think, after all this time, SegWit has still not been activated by miners (where "miners" means essentially the 2 or 3 largest in the business which control the majority of hashing power)? 

These miners have "agreed" to the so-called Barry Silbert Accords from Consensus 2017. But I would be very surprised if push came to shove and those miners actually activated SegWit SF as [that plan](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-March/013921.html) calls for. They have been stalling for nearly 2 years now, dragging the core developers along with them. 

There is no reason to be opposed to SegWit at this point as it's already been activated on Litecoin and the technical FUD has been debunked. SegWit effectively gives us bigger blocks (i.e. more transactions per block), something the miners point to as the reason they have been clamoring for an increased 2MB blocksize limit. 

I'm amazed that @luke-jr and @achow101 took the time out of their respective days to even respond to this piece of sh*t trolling effort.